board/opentrons/ot2/rootfs-overlay/etc/nginx/nginx.conf

```diff
@@ -51,5 +51,23 @@ http {
             proxy_request_buffering  off;
             client_max_body_size     0;  # Unlimited to allow large update files.
         }
+
+        location /system {
+            proxy_http_version       1.1;
+            proxy_read_timeout       1h;
+            proxy_pass               http://127.0.0.1:32950;
+        }
+
+        # The /system/time endpoints live in the robot server, and thus
```

Comment:
I think you can nest locations in nginx config, which would maybe let you not copy/paste this stuff

Reply:
Yep that worked. Still have to duplicate some of the settings because the only one here that gets duplicated across levels is `proxy_set_header`